tensorboard/webapp/feature_flag/effects/feature_flag_effects.ts

```diff
@@ -95,6 +98,32 @@ export class FeatureFlagEffects {
     {dispatch: false}
   );
 
+  readonly resetFeatureFlagOverrides$ = createEffect(
+    () =>
+      this.actions$.pipe(
+        ofType(resetFeatureFlagOverrides),
+        tap(({flags}) => {
+          flags.forEach((flag) => {
+            this.dataSource.resetPersistedFeatureFlag(
+              flag as keyof FeatureFlags
```

Comment:
This type casting does not seem necessary. The action defines this already.  Obviously if the compiler is complaining then I guess we have to leave it. I just wanted to make sure this was the case.

Reply:
Good call, I think this was necessary in an older version.